### DISCUSSION TITLE
duplicate default directory for new viewer apps

### DISCUSSION THREAD

**Discussion by JeffPollock** [iTwin/itwinjs-core#4806] [Q&A] [answered]
After posting: npx create-react-app your-app-name --template @itwin/web-viewer --scripts-version @bentley/react-scripts in Visual Studio, I get this response:

The directory your-app-name contains files that could conflict:

  .env
  node_modules/
  package-lock.json
  package.json
  public/
  src/
  tsconfig.json

Either try using a new directory name, or remove the files listed above.

Any suggestions about how to resolve this?

**Comment by aruniverse** [comment, marked as answer]
Do you already have a directory called `your-app-name`?

**Reply by JeffPollock** [reply]
That could be it!  I think I created it previously at an earlier time when I was trying to set up.  

Thank you Arun!
